languages.toml

```
@@ -880,6 +880,31 @@ indent = { tab-width = 4, unit = "    " }
 name = "php"
 source = { git = "https://github.com/tree-sitter/tree-sitter-php", rev = "f860e598194f4a71747f91789bf536b393ad4a56" }
 
+[[language]]
+name = "php_only"
+scope = "source.php_only"
+injection-regex = "php_only"
+file-types = ["blade"]
+shebangs = ["php_only"]
+roots = ["composer.json", "index.php"]
+language-servers = [ ]
+indent = { tab-width = 4, unit = "    " }
```

This language can be "hidden" since it's only used by injections:

```suggestion
name = "php-only"
scope = "source.php-only"
file-types = []
indent = { tab-width = 4, unit = "    " }
```